4주차/[Lv.2]_게임_맵_최단거리/이상훈.js

```diff
@@ -0,0 +1,40 @@
+const isInBoundary = (maps, y, x) => {
+  if (y < 0 || y >= maps.length) return false;
+  if (x < 0 || x >= maps[0].length) return false;
+  if (!maps[y][x]) return false;
+
+  return true;
+};
+
+function solution(maps) {
+  const queue = [];
+  const visited = Array.from({ length: maps.length }, () => Array.from({ length: maps[0].length }).fill(0));
+  const [targetY, targetX] = [maps.length - 1, maps[0].length - 1];
+
+  queue.push([0, 0, 1]);
```

Comment:
```suggestion
  const queue = [[0, 0, 1]];
  const visited = Array.from({ length: maps.length }, () => Array.from({ length: maps[0].length }).fill(0));
  const [targetY, targetX] = [maps.length - 1, maps[0].length - 1];
```
`push()`를 생략할 수 있을 것 같습니다!! 🙋🏻‍♂️